scss/previews/_examples.scss

```diff
@@ -23,19 +15,44 @@
   }
 }
 
-[data-sassdoc='example-tab-solo'] {
-  @include contrasted('text');
-  display: inline-block;
-  padding: 0 size('shim');
+[data-sassdoc='example-panel'] {
+  align-items: stretch;
+  border: 1px solid color('border');
+  display: flex;
+  flex-wrap: wrap;
+
+  &[aria-hidden='true'] {
+    display: none;
+  }
 }
 
-[data-sassdoc-example-panel] {
-  border: 1px solid color('border');
+[data-sassdoc-example] {
+  align-items: stretch;
+  display: flex;
+  flex: 1 1 40%;
+  flex-direction: column;
   max-height: size('rhythm') * 12;
   max-width: 100%;
+  min-width: 20em;
   overflow: auto;
+
+  & + & {
+    border-left: 1px solid color('border');
+  }
+}
+
+[data-sassdoc-title] {
+  @include contrasted('light-gray');
+  @include font-family('code');
+  @include font-size('smaller');
+  flex: 0 0 auto;
+  padding: 0 size('shim');
+
+  + [data-sassdoc] {
+    flex: 1 1 auto;
+  }
 }
 
-[data-sassdoc-example-panel='output'] {
+[data-sassdoc-rendered] {
   padding: size('shim');
```

Comment:
This makes the rendered panel more narrow than the tab groups. We can eliminate l/r padding. Maybe in favor of using t/b margins instead?